### PR TITLE
New fix for #27

### PR DESCRIPTION
**Pull request by Pimprenelle9**
Hello,

Even with the last version of the plugin, I have the same issue as #27.
Digging into the CSS, my team and I found out that it was just caused by the use of "visibility" instead of "display".

On both my production environment and demonstrator, this fixes the issue.

### CODE REVIEW

**Comment by jhildenbiddle** [comment]
Hi @Pimprenelle9 --

The reason why `visibility: [hidden|visible];` is used instead of `disable: [none|block]` is so hidden tab content has dimensions default. This is often necessary for content that is initialized via JavaScript such as embeds from places like CodePen, CodeSandbox, etc.

More importantly, I can reproduce the bug now. The reason I wasn't seeing it before is because I use [docsify-themeable](https://jhildenbiddle.github.io/docsify-themeable/) for all of my docsify sites. As mentioned in #27, the issue only presents itself when using one of docsify's default themes like `vue.css`.

TLDR; swapping `visibility` for `display` will solve one problem while introducing another. Let me dig into this and see what I can do. If you've got the time, feel free to take a crack at it as well. :)

**Comment by jhildenbiddle** [comment]
Hi @Pimprenelle9. This has been fixed. See details here: https://github.com/jhildenbiddle/docsify-tabs/issues/27#issuecomment-1242745240

Thank you for taking the time to file the PR. Much appreciated!
